src/panoramic/cli/command.py

```diff
@@ -291,6 +288,7 @@ def detect_joins(target_dataset: Optional[str] = None, diff: bool = False, overw
 
     if local_state.is_empty:
         echo_info('No datasets to detect joins on')
+        # TODO: should we return here?
```

Comment:
Yeah, good spot - this is missing a `return` for sure - I think we can add it and if it doesn't break tests we should be good to go.

Reply:
Added the return in 07d5fd42df578ce45b4630276102c7e926fc8559